core/domain/story_services.py

```diff
@@ -605,6 +605,28 @@ def get_completed_nodes_in_story(user_id, story_id):
     return completed_nodes
 
 
+def get_pending_nodes_in_story(user_id, story_id):
+    """Returns the nodes that are pending in a story
+
+    Args:
+        user_id: str. The user id of the user.
+        story_id: str. The id of the story.
+
+    Returns:
+        list(StoryNode): The list of the story nodes that are pending
+        of an user.
```

Comment:
The list of story nodes, pending for the user.

Reply:
Hi @aks681, changes done. PTAL. Thanks!